modules/runtime/pages/bonita-applications-interface-overview.adoc

```diff
@@ -22,9 +22,9 @@ Bonita Runtime embeds two types of applications:
   ** _User_ , based on a standard BPM usage, centered on a generic tasklist
   ** _Administrator_, allowing the management (monitoring and troubleshooting) of an automation project in production (BDM excepted)
 
-These applications are installed by default for a fresh installation of the platform. +
-In addition to that, the generic Bonita Applications can be downloaded from the Welcome Page of the Studio, in the "Resources" tile. +
-You may want to do so to create your applications from an example. +
+Core applications are installed by default and cannot be removed.
+Generic applications will be installed on a fresh installation. They can also be downloaded from the Welcome Page of the Studio, in the "Resources" tile. +
+You may want to do so to create your applications from an example, or to include these applications in your self-contained bonita application (see: xref::application-build.adoc)
```

Comment:
Invalid xref. 
Should be xref:runtime:application-build.adoc
See https://github.com/bonitasoft/bonita-documentation-site/blob/master/docs/content/CONTRIBUTING.adoc#adding-reference-between-pages-of-two-components-or-between-pages-of-2-versions-of-the-same-component

Reply:
done :heavy_check_mark: